pages/_app/route.js

```diff
@@ -0,0 +1,24 @@
+import Route from 'components/application/route'
+
+/**
+ * NextRoute is a Route specific for the Next.js. It defines `href` and `as`
+ * properties according to the `next/router` specification.
+ */
+class NextRoute extends Route {
```

Comment:
how is this used?

Reply:
I App to handle `href` and `as` parameters for router